### ISSUE TITLE
[Bug] Docs incorrect about units for "interval"

### ISSUE THREAD

**Issue by chrisjshull** [ztalbot2000/homebridge-cmd4#139]
https://ztalbot2000.github.io/homebridge-cmd4/autoGenerated/CMD4_AccessoryDescriptions.html#interval_D and https://ztalbot2000.github.io/homebridge-cmd4/autoGenerated/CMD4_AccessoryDescriptions.html#interval both claim that "interval" is in seconds. Is it actually in milliseconds?

**Comment by chrisjshull** [comment]
also, apparently the docs deep linking doesn't work - those are supposed to be the links to the Directive and Accessory Directive pages on "interval"

**Comment by ztalbot2000** [comment]
Hi,

  I will have a look.  If I remember correctly, interval is in seconds, but
it's always good to check again.

Ttyl,
John

**Comment by ztalbot2000** [comment]
Hi,

   Sorry, I don't have much time lately.  The interval is actually in seconds. If you look at Cmd4Accessory.js under the parseConfig procedure in the current release on line 1581 you will see the line:

case constants.INTERVAL:
               // Intervals are in seconds
               this.interval = parseInt( value, 10 ) * 1000;

As for the deep link. Under Directives interval does link to:
https://ztalbot2000.github.io/homebridge-cmd4/autoGenerated/CMD4_AccessoryDescriptions.html#interval_D
For whatever reason on Safari, I have to scroll up one line to see the header.
I did add link checks to V7.0.1 and while it did not find anything related to your bug, it did find two for tags that were moved to the Developers documentation. So Thanks for making me double check it all.
I'll be releasing 7.0.1 momentarily.  The documentation has already been submitted.

John
